python/python/tests/test_async_client.py

```diff
@@ -2233,6 +2279,23 @@ async def test_cluster_fail_routing_by_address_if_no_port_is_provided(
             await redis_client.info(route=ByAddressRoute("foo"))
 
 
+@pytest.mark.asyncio
+class TestExceptions:
+    @pytest.mark.parametrize("cluster_mode", [True, False])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_timeout_exception_with_blpop(self, redis_client: TRedisClient):
+        key = get_random_string(10)
+        with pytest.raises(TimeoutError):
```

Comment:
should not raise an error anymore with any timeout AFAIK
I want to add such test for all blocking commands